### ISSUE TITLE
README not being detected, even though it exists in github repo (Sadly this package has no readme)

### ISSUE THREAD

**Issue by eddyerburgh** [npm/registry-issue-archive#118]
I just published a patch version of my package. The repo has a [README](https://github.com/eddyerburgh/avoriaz/blob/master/README.md), but  my [NPM package page](https://www.npmjs.com/package/avoriaz) can't find one:

```
Sadly, this package has no readme. Go write one!
```

local branch is up-to-date with remote repository.

Any ideas?

**Comment by lennym** [comment]
I'm seeing this on (nearly) all of my modules as well as a lot of other modules. I don't seem to be able to find any commonality around modules where the README appears or not.

The only vague pattern I can find seems to be that *very* popular packages all seem to be rendering READMEs correctly. That isn't all that reliable though, as the only one of my packages that is rendering correctly is npmjs.com/choma, which basically never gets downloaded.

Edit: it's not even consistent for popular packages - https://www.npmjs.com/package/yo is from the list of popular packages on the homepage, and doesn't show a README.

**Comment by soldair** [comment]
http://status.npmjs.org/incidents/tj3qsythtgg8 here is the status page incident please check it for updates

**Comment by soldair** [comment]
this is now resolved. we are monitoring and ill reopen if we find its still an issue

**Comment by erikerikson** [comment]
This issue is showing at:
https://www.npmjs.com/package/serverless

**Comment by erikerikson** [comment]
![image](https://cloud.githubusercontent.com/assets/571200/22389896/e5e3e28a-e49c-11e6-83c7-e124480f1524.png)

**Comment by erikerikson** [comment]
![image](https://cloud.githubusercontent.com/assets/571200/22389917/02097092-e49d-11e6-9115-a2475a10c3aa.png)

**Comment by ng-state** [comment]
Is this issue resolved? Because Readme cannot be detected at: https://www.npmjs.com/package/ng-state
but it exists in github.

**Comment by erikerikson** [comment]
@ng-state you may want to read this comment https://github.com/npm/registry/issues/120#issuecomment-276157329.  It resolved the matter for us.
tl;dr: update npm and publish again

**Comment by erikerikson** [comment]
tl;dr: update npm and publish again

**Comment by edertone** [comment]
Turbocommons library has the same problem:
https://www.npmjs.com/package/turbocommons-ts

and the turbocommons git repo has a readme:
https://github.com/edertone/TurboCommons

Don't know why it is not showing on npm...

**Comment by soldair** [comment]
https://github.com/edertone/TurboCommons/tree/master/TurboCommons-TS
doesn't have it own readme. The readme must be published in the tarball as part of the package for npm to render it. @edertone

**Comment by edertone** [comment]
I thought that when an npm package has no included readme, the one that is located at the specified github page would be used instead... Am I wrong with this assumption?

**Comment by soldair** [comment]
yes. npm has never used any github apis to render readmes. its always been the version of the readme published with the package when the published package is the latest version.

**Comment by edertone** [comment]
Thanks, thats what I suspected. So puting a readme.md file at the root where the package.json lies will solve the problem? Or do i need to add some entry to the package.json?

**Comment by soldair** [comment]
yeah. at the root next to the package json

**Comment by lararosekelley** [comment]
Just had this issue today. Attempted to publish a package (have done this plenty of times in the past) and no README is showing up. Have never had this issue before. 99% sure I'm not doing anything wrong in the package.json:

1. GitHub repo - https://github.com/greylocklabs/teapot
2. npm package - https://www.npmjs.com/package/node-teapot

I think this is a registry issue, `npm info` recognizes the file, and if you install the package the README file is there as well. Using Node 8.9.4 with npm 5.6.0

**Comment by lararosekelley** [comment]
Hmm, showing up now. Probably just being too impatient!

**Comment by olore** [comment]
Impatience got me too! Took all of 5 minutes  😊

**Comment by Procaseycash** [comment]
No readme found even when there is readme on the github repo (https://github.com/Procaseycash/ng-pure-datatable)
https://www.npmjs.com/package/ng-pure-datatable

But I can find the readme at https://npm.runkit.com/ng-pure-datatable

**Comment by drihu** [comment]
No readme found!
It has a readme on the github repo
github repo: [https://github.com/drihu/poketask](https://github.com/drihu/poketask)
npm package: [https://www.npmjs.com/package/poketask](https://www.npmjs.com/package/poketask)

**Comment by drihu** [comment]
I republished my package, now npm shows it :/

**Comment by Mouvedia** [comment]
It seems to be a recurrent problem: one of my modules <del>is</del><ins>was</ins> affected.
